core/lexicon/en/system_events.inc.php

```diff
@@ -12,4 +12,15 @@
 $_lang['error_log_download'] = 'Download Error Log ([[+size]])';
 $_lang['error_log_too_large'] = 'The error log at <em>[[+name]]</em> is too large to be viewed. You can download it via the button below.';
 $_lang['system_events'] = 'System Events';
-$_lang['priority'] = 'Priority';
\ No newline at end of file
+$_lang['priority'] = 'Priority';
+
+$_lang['deprecated_log'] = 'Deprecations Log';
+$_lang['deprecated_log_desc'] = 'From time to time, old functionality is marked as deprecated to be removed in a future version. In such cases you\'ll find deprecation notices that alert you that updates will be necessary in the future.';
```

Comment:
```suggestion
$_lang['deprecated_log_desc'] = 'From time to time, old functionality is marked as deprecated and to be removed in a future version. In such cases you\'ll find deprecation notices that alert you that updates will be necessary in the future.';
```

Reply:
I think "marked as deprecated to be removed in a future version" is the correct English here: "deprecated to be removed" being the **one** status it is given. If native speakers disagree please reopen the conversation.